website/content/docs/enterprise/network-segments.mdx

```diff
@@ -11,19 +11,199 @@ description: |-
 <EnterpriseAlert>
   This feature requires{' '}
   <a href="https://www.hashicorp.com/products/consul/">Consul Enterprise</a>{' '}
-  with the Global Visibility, Routing, and Scale module.
+  version 0.9.3+ with the Global Visibility, Routing, and Scale module.
 </EnterpriseAlert>
 
-Consul Network Segments enables operators to create separate LAN gossip segments
-in one Consul cluster. Agents in a segment are only able to join and communicate
-with other agents in its network segment. This functionality is useful for
-clusters that have multiple tenants that should not be able to communicate
-with each other.
+Consul requires full connectivity between all agents (servers and clients) in a
+datacenter within a given LAN gossip pool. By default, all Consul agents will be
```

Comment:
@blake : do we refer to this as a "datacenter" or a "cluster" now? I ask because under the Consul Networking Models section, we use "Cluster". Interestingly, Cluster isn't a recognized term in the [glossary](https://www.consul.io/docs/install/glossary#datacenter), but Datacenter is.

Reply:
Agents are configured to be members of a specific [datacenter](https://www.consul.io/docs/agent/options#_datacenter). By default, within a given datacenter, Consul requires full LAN gossip connectivity between all agents.